source/server/BUILD

```diff
@@ -462,9 +461,10 @@ envoy_cc_library(
         "//source/common/signal:fatal_error_handler_lib",
         "//source/common/singleton:manager_impl_lib",
         "//source/common/stats:thread_local_store_lib",
+        "//source/common/tls:context_lib",
         "//source/common/upstream:cluster_manager_lib",
-        "//source/common/upstream:health_discovery_service_lib",
```

Comment:
I think this deletion was accidental, and you meant to remove tls:config two lines down, right?